server/test/index.test.js

```diff
@@ -0,0 +1,18 @@
+const request = require('supertest');
+const app = require('../app');
+const dbBuild = require('../database/config/build');
+const connection = require('../database/config/connection');
+
+beforeEach(() => dbBuild());
+afterAll(() => connection.end());
+
+describe('Get all patients', () => {
+  test('Route /patients status 200, json header', async () => {
+    expect.assertions(1);
+    const res = await request(app)
+      .get('/api/v1/patients')
+      .expect(200)
+      .expect('Content-Type', /json/);
+    expect(res.body.data).toHaveLength(20);
```

Comment:
you can add return here before expect like this and remove the expect.assertions(1);
```js
return expect(res.body.data).toHaveLength(20);
```